style.css

```diff
@@ -97,8 +100,170 @@ nav .menu-bar {
   list-style: none;
   padding: 0;
   display: flex;
+  margin-bottom: 12px;
 }
 
 .icons li {
   margin-right: 24px;
 }
+
+.portfolio-list-section {
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  width: 327px;
+  padding: 12px;
+}
```

Comment:
- [ ] The [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) guide us to use `CSS flexbox` in order to layout the individual card.  I would to ensure laying the contents of each card with [flexbox](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwja3pOm4sf8AhWT97sIHaqxDJoQFnoECAsQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fcss%2Fcss3_flexbox.asp&usg=AOvVaw3m0MpAK8nDKuWZA9cVp4bC) :walking: